### PR TITLE
Updated github user docs

### PR DESCRIPTION
**Pull request by danielimprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Updated the github userdocs

#### Primary reviewers
@m-samiec @oblm @ElleEss @anne-edwards

### CODE REVIEW

File: SpatialGDK/Documentation/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# The SpatialOS GDK for Unreal (pre-alpha) documentation on GitHub
+# The SpatialOS GDK for Unreal (alpha) documentation on GitHub
 
 This documentation on GitHub is a copy of the [GDK documentation on the SpatialOS documentation website](https://docs.improbable.io/unreal/pre-alpha). Due to this, the internal links do not work and the image links are broken.
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/README.md]: Are the commented out comments in the line of the rest of the files needed? @ElleEss

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ later releases and we’ll keep you posted on this). However, we are taking issu
 See the [GDK for Unreal C++ coding standards guide]({{urlRoot}}/contributions/unreal-gdk-coding-standards).
 
 ## Warning
-This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)
\ No newline at end of file
+This [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)
```
Comment (girayimprobable) [inline CONTRIBUTING.md]: We need to remove the "evaluation and feedback purposes". Taken from README:

```
This is an alpha (SpatialOS website) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback - see the guidance on Recommended use, below.
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+
+The GDK offers:
+* **Multi-server support**: leveraging our cloud platform SpatialOS (SpatialOS documentation), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
+* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.
+* **Easy getting started**: we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.
```
Comment (girayimprobable) [inline README.md]: "Easy to get started?", no? @ElleEss

File: README.md [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
+The SpatialOS Games Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.
+
+The GDK offers:
+* **Multi-server support**: leveraging our cloud platform SpatialOS (SpatialOS documentation), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
+* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.
```
Comment (girayimprobable) [inline README.md]: Technically there is no "character movement controller"

We can just say "character movement". Also can add stuff like ability system maybe, although it doesn't work across boundaries yet.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -8,4 +8,4 @@ later releases and we’ll keep you posted on this). However, we are taking issu
 See the [GDK for Unreal C++ coding standards guide]({{urlRoot}}/contributions/unreal-gdk-coding-standards).
 
 ## Warning
-This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)
\ No newline at end of file
+This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback - see the guidance on Recommended use, below.
```
Comment (anne-edwards) [inline CONTRIBUTING.md]:
```suggestion
This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal, pending stability and performance improvements. The API may change as we learn from feedback - see the guidance on [Recommended use]({{urlRoot}}/readme/recommended-use).
```

There's no "recommend use" section below on this page, so I tried adding a link to the one in the README.md instead. Not sure if the link is right though.

File: SpatialGDK/Documentation/README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/README.md]:
```suggestion
This documentation on GitHub is a copy of the [GDK documentation on the SpatialOS documentation website](https://docs.improbable.io/unreal/latest). Due to this, the internal links do not work and the image links are broken. 
```

Changed "pre-alpha" to "latest" - I think that's right?

File: README.md [hunk 7]
```diff
@@ -1,11 +1,16 @@
 # The SpatialOS Games Development Kit for Unreal (alpha)
->This is an [alpha (SpatialOS website)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
+The SpatialOS Games Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.
+
+The GDK offers:
+* **Multi-server support**: leveraging our cloud platform SpatialOS (SpatialOS documentation), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
```
Comment (anne-edwards) [inline README.md]:
```suggestion
* **Multi-server support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one, enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
```

File: README.md [hunk 8]
```diff
@@ -1,11 +1,16 @@
...
+* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement, client-side prediction.
```
Comment (anne-edwards) [inline README.md]:
```suggestion
* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement, and client-side prediction.
```

File: README.md (hunk 3 above)
Comment (anne-edwards) [reply README.md]: Perhaps "An easy onboarding experience"? Or maybe "An easy experience getting started"?

File: README.md [hunk 9]
```diff
@@ -1,11 +1,16 @@
...
+* **Easy getting started**: we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.
 
-The SpatialOS Games Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) made by [Improbable](. You can use the GDK to make use of multiple Unreal servers in a single game instance using the SpatialOS platform, unlocking the ability to implement large-scale, complex and persistent worlds. Find out more about the GDK in [our blogpost](https://improbable.io/games/blog/spatialos-unreal-gdk-pre-alpha).
+>This is an [alpha (SpatialOS website)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
```
Comment (anne-edwards) [inline README.md]:
```suggestion
>This is an [alpha (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
```

File: README.md [hunk 10]
```diff
@@ -1,11 +1,16 @@
...
 
 ----
-* [Installation and setup](https://docs.improbable.io/unreal/latest/setup-and-installing) (on the documentation website)
-* [Documentation](https://docs.improbable.io/unreal/alpha/index) (on the documentation website)
+* [Get started](https://docs.improbable.io/unreal/latest/content/get-started/introduction) (on the documentation website)
```
Comment (anne-edwards) [inline README.md]:
```suggestion
* [Get started](https://docs.improbable.io/unreal/latest/content/get-started/introduction) (on the SpatialOS documentation website)
```

File: README.md [hunk 11]
```diff
@@ -1,11 +1,16 @@
...
+* [Documentation](https://docs.improbable.io/unreal/latest) (on the documentation website)
```
Comment (anne-edwards) [inline README.md]:
```suggestion
* [Documentation](https://docs.improbable.io/unreal/latest) (on the SpatialOS documentation website)
```

File: README.md [hunk 12]
```diff
@@ -37,7 +42,7 @@ We have released the GDK for Unreal this early in development because we want yo
 The GDK and its starter projects are available on GitHub.
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
 * [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
-* [Third Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
 * [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
 * [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) </br>
```
Comment (anne-edwards) [inline README.md]: For consistency with my comments in the other README docs, we could move this up so it's just below the GDK bullet point. (Or, if you didn't make that change in the other docs, leave it here as well.)

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+
+We are currently not accepting public contributions (but we do plan to in 
+later releases and we’ll keep you posted on this). However, we are taking issues and we do want your 
+[feedback](./README.md#give-us-feedback).
+
+## Coding standards
+See the [GDK for Unreal C++ coding standards guide](./SpatialGDK/Documentation/contributions/unreal-gdk-coding-standards.md).
+
+## Warning
+This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback - see the guidance on [Recommended use](https://docs.improbable.io/unreal/latest/content/recommended-use).
```
Comment (ElleEss) [inline CONTRIBUTING.md]: SpatialOS GDK for Unreal

File: README.md [hunk 14]
```diff
@@ -1,11 +1,16 @@
-# The SpatialOS Games Development Kit for Unreal (alpha)
->This is an [alpha (SpatialOS website)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
+# The SpatialOS Game Development Kit for Unreal (alpha)
+The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.
```
Comment (ElleEss) [inline README.md]: [plugin (Unreal documentation)]

File: README.md [hunk 15]
```diff
@@ -1,11 +1,16 @@
...
 
-The SpatialOS Games Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) made by [Improbable](. You can use the GDK to make use of multiple Unreal servers in a single game instance using the SpatialOS platform, unlocking the ability to implement large-scale, complex and persistent worlds. Find out more about the GDK in [our blogpost](https://improbable.io/games/blog/spatialos-unreal-gdk-pre-alpha).
+The GDK offers:
+* **Multi-server support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
```
Comment (ElleEss) [inline README.md]: one,  enabling
(Double space)

File: README.md [hunk 16]
```diff
@@ -35,11 +40,11 @@ We have released the GDK for Unreal this early in development because we want yo
...
 The GDK and its starter projects are available on GitHub.
+* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
 * [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
-* [Third Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
-* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
-* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) </br>
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
+* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)</br>
 **NOTE:** This link may give you a 404.
```
Comment (joshuahuburn) [inline README.md]: This section referring to `This link may give you a 404.` is incorrect. This is only for the UnrealEngine fork which is already stated.

File: README.md [hunk 17]
```diff
@@ -1,11 +1,16 @@
...
+* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement, and client-side prediction.
+* **An easy onboarding experience**: we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.
+
+>This is an [alpha (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
```
Comment (ElleEss) [inline README.md]: SpatialOS Unreal GDK
->
SpatialOS GDK for Unreal

File: README.md [hunk 18]
```diff
@@ -50,5 +55,4 @@ In order to get access to this fork, you need to link your GitHub account to a v
 * Version: alpha (stability and performance improvements pending)
 * GDK repository: [github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
```
Comment (joshuahuburn) [inline README.md]: GDK repository is now listed above so not needed here.

File: README.md [hunk 19]
```diff
@@ -36,10 +41,10 @@ We have released the GDK for Unreal this early in development because we want yo
 ## Where to get the GDK and starter projects
```
Comment (ElleEss) [inline README.md]: starter projects .<- Didn't you want to rename this title to something which didn't say "starter project"?

File: SpatialGDK/Documentation/welcome.md [hunk 20]
```diff
@@ -0,0 +1,18 @@
...
+* **An Unreal-native experience:** keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features which Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.
+* **An easy way to get started:** we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.
+
+#### [Get started]({{urlRoot}}/content/get-started/introduction.md) with the SpatialOS GDK for Unreal.
+
+<%(Callout type="warn" message="This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use]({{URLRoot}}/content/recommended-use).")%>
```
Comment (joshuahuburn) [inline SpatialGDK/Documentation/welcome.md]: Do we still need this callout? Thought we nuked them.

File: README.md [hunk 21]
```diff
@@ -36,10 +41,10 @@ We have released the GDK for Unreal this early in development because we want yo
...
 The GDK and its starter projects are available on GitHub.
```
Comment (ElleEss) [inline README.md]: starter projects -> sample projects

File: README.md (hunk 19 above)
Comment (ElleEss) [reply README.md]: sample projects

File: README.md [hunk 22]
```diff
@@ -36,10 +41,10 @@ We have released the GDK for Unreal this early in development because we want yo
...
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
 * [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
-* [Third Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
-* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
-* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) </br>
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
```
Comment (ElleEss) [inline README.md]: Game -> game

File: README.md [hunk 23]
```diff
@@ -36,10 +41,10 @@ We have released the GDK for Unreal this early in development because we want yo
...
+* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)</br>
 **NOTE:** This link may give you a 404.
```
Comment (ElleEss) [inline README.md]: This note needs to sit under the UE4 link - it's why it was last before - I think Giray re-ordered but the 404 note doesn't make sense now, I have just realised.

File: SpatialGDK/Documentation/welcome.md [hunk 24]
```diff
@@ -0,0 +1,18 @@
...
+<img src="{{assetRoot}}assets/unrealgdk-headline-image.png" style=" margin: 0 auto; display: block; width: 60%" />
+
+<br/>
+
+<%(TOC)%>
+
+The SpatialOS Game Development Kit (GDK) for Unreal is a <a href="" data-track-link="Plugin Link Clicked|product=Docs|platform=Win|label=Win" target="_blank">plugin (Unreal documentation)</a>  which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API. 
+
+The GDK offers:
+* **Multi-server support:** leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/welcome.md]: Multiserver
(This is what Unreal does)

File: SpatialGDK/Documentation/index.md [hunk 25]
```diff
@@ -13,6 +13,6 @@ The GDK offers:
 * **An Unreal-native experience:** keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features which Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/index.md]: Need to standardize onto "mulitserver". (No hyphen.)

File: README.md (hunk 15 above)
Comment (girayimprobable) [inline README.md]: We say "single-authority engine into distributed" twice in this section, let's change this one slightly. 

```leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.``

File: SpatialGDK/Documentation/welcome.md (hunk 20 above)
Comment (danielimprobable) [reply SpatialGDK/Documentation/welcome.md]: Yep I think this is the main place where we do such a callout

File: README.md (hunk 23 above)
Comment (girayimprobable) [reply README.md]: Yes

File: SpatialGDK/Documentation/welcome.md (hunk 20 above)
Comment (girayimprobable) [reply SpatialGDK/Documentation/welcome.md]: Yes

File: README.md (hunk 18 above)
Comment (danielimprobable) [reply README.md]: I think we might as well have it here

File: SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md [hunk 26]
```diff
@@ -115,7 +115,7 @@ etc...
     >
     > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config files under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
     >
-    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
+    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multiserver features of the GDK, which may lead to erratic behavior.
```
Comment (joshuahuburn) [inline SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md]: Please revert this change. This whole doc is being nuked and replaced with the new one https://github.com/spatialos/UnrealGDK/pull/427

File: README.md [hunk 27]
```diff
@@ -33,14 +38,15 @@ In order to transform Unreal from a single server engine to a distributed model,
 ## Give us feedback
 We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).
 
-## Where to get the GDK and starter projects
-The GDK and its starter projects are available on GitHub.
+## Where to get the GDK and example projects
```
Comment (girayimprobable) [inline README.md]: Should we say "related projects" instead? Some of these are not really example projects.

File: README.md (hunk 18 above)
Comment (ElleEss) [reply README.md]: It's there for form as you usually have your repo URL on your repo readme so you can find it again when you clone this baby.

**Comment by joshuahuburn** [comment]
Great job reviewing this PR @m-samiec
